xml/cap_admin_backup-restore.xml

```diff
@@ -521,5 +523,249 @@
+     </para>
+    <screen>&prompt.user;kubectl cp <replaceable>/tmp/blobstore-src.tgz</replaceable> scf/blobstore-0:/</screen>
+    </step>
+    <step>
+     <para>
+      Restore the contents of the archive created during the backup process to the <literal>blobstore</literal> pod:
+     </para>
+<screen>&prompt.user;kubectl exec -it --namespace scf blobstore-0 -- bash -l -c 'monit stop all &amp; sleep 10 &amp; rm -rf /var/vcap/store/shared/* &amp; tar xvf blobstore-src.tgz &amp; monit start all &amp; rm blobstore-src.tgz'</screen>
+    </step>
+    <step>
+     <para>
+      Recreate the CCDB on the <literal>mysql</literal> pod:
+     </para>
+<screen>&prompt.user;kubectl exec mysql-0 --namespace scf -- bash -c \
+    "/var/vcap/packages/mariadb/bin/mysql \
+    --defaults-file=/var/vcap/jobs/mysql/config/mylogin.cnf \
+    -e 'drop database ccdb; create database ccdb;'"</screen>
+    </step>
+    <step>
+     <para>
+      Restore the CCDB on the <literal>mysql</literal> pod:
+     </para>
+     <screen>&prompt.user;kubectl exec -i mysql-0 --namespace scf -- bash -c '/var/vcap/packages/mariadb/bin/mysql --defaults-file=/var/vcap/jobs/mysql/config/mylogin.cnf ccdb' &lt; /tmp/ccdb-src.sql</screen>
+    </step>
+    <step>
+     <para>
+      Start the monit services on the <literal>api-group-0</literal>, <literal>cc-worker-0</literal>, and <literal>cc-clock-0</literal> pods
+     </para>
+<screen>&prompt.user;for n in api-group-0 cc-worker-0 cc-clock-0; do
+  kubectl exec -it --namespace scf $n -- bash -l -c 'monit start all'
+done
+</screen>
+    </step>
+    <step>
+     <para>
+      If your old cluster did not have <literal>current_key_label</literal> defined, perform a key rotation. Otherwise, a key rotation is not necessary.
+     </para>
+     <substeps>
+      <step>
+       <para>
+        Change the encryption key in the config file:
+       </para>
+      <screen>&prompt.user;kubectl exec --namespace scf api-group-0 -- bash -c 'sed -i "/db_encryption_key:/c\\db_encryption_key: \"$(echo $CC_DB_ENCRYPTION_KEYS | jq -r .migrated_key)\"" /var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml'
+</screen>
+      </step>
+      <step>
+       <para>
+        Run the rotation for the encryption keys:
+       </para>
+       <screen>&prompt.user;kubectl exec --namespace scf api-group-0 -- bash -c 'export PATH=/var/vcap/packages/ruby-2.4/bin:$PATH ; export CLOUD_CONTROLLER_NG_CONFIG=/var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml ; cd /var/vcap/packages/cloud_controller_ng/cloud_controller_ng ; /var/vcap/packages/ruby-2.4/bin/bundle exec rake rotate_cc_database_key:perform'</screen>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>
+      The data restore is now complete. Run commands such as <command>cf apps</command>, <command>cf marketplace</command>, <command>cf services</command>, etc. and verify data from the old cluster is returned.
+     </para>
```

Comment:
Don't use "etc." We do not handwave, we provide specifics. "Run some cf commands, such as cf apps, cf marketplace, or cf services, and verify data from the old cluster is returned."

Reply:
Addressed in https://github.com/SUSE/doc-cap/pull/280/commits/fcde67d3da1a19b5743da5d0fb77df36f5e37f01